src/store/modules/picklist/index.ts

```diff
@@ -9,11 +9,11 @@ const picklistModule: Module<PicklistState, RootState> = {
   namespaced: true,
   state: {
     current: {},
-    list: []
+    list: {}
```

Comment:
Why we have set initial value to {} instead of list?

Reply:
Updated it sir